### PR TITLE
[EN-3830] [SF85P] Sections 1-5

### PR DESCRIPTION
**Pull request by christopherhuii**
## Description
This PR updates sections 1-5 to fully support the SF85P. The changes were made in accordance to the [form tracking doc](https://docs.google.com/spreadsheets/d/1nHJhOiQFH2k0QzQQ0HkrsllDWh6ED5yi4ytbioiLjNs/edit#gid=889400045). **You can access the SF85P with the URL param `?formType=SF85P`.**

- updates to the instructions copy. please review this thoroughly. most of the text was copy and pasted from previous forms, but i referenced the SF85P content guide to make sure form specific things are in their correct places @michaelccata @joshmfranklin 

| Section | Tracking Doc Line | Notes |
|---------|------------------- |-------|
| Where you have lived | 86-106 | 7 years requirement |
| Employment activities | 122-198 | 7 years requirement |
| Where you went to school | 107 - 121 | 7 years requirement |
| Marital | 262 | doc expiration not required |
| Marital | 285 | phone number not required |
| Cohabitants | 299 | doc expiration not required |
| Relatives | 323 - 345 | some complicated branching, but it was mostly done from SF85 |
| Dual/Multiple Citizenships | 72 | renounce citizenship not required (should be done from SF85) |
| Military | 239 - 245 | maintains contact not required (should be done from SF85) "

Note: Don't merge until #1759 is merged
## Checklist for Reviewer

- [ ] Review code changes
- [ ] Review changes for Database effects
- [ ] Verify change works in IE browser

More details about this can be found in [docs/review.md](docs/review.md)

### CODE REVIEW

File: src/models/foreignBornDocument.js [hunk 1]
```diff
@@ -18,9 +18,12 @@ const foreignBornDocument = {
...
     return {}
   },
-  DocumentExpiration: (value, attributes) => {
-    if (attributes.DocumentExpirationNotApplicable
-      && attributes.DocumentExpirationNotApplicable.applicable === false) {
+  DocumentExpiration: (value, attributes, attributeName, options) => {
+    if (
+      (attributes.DocumentExpirationNotApplicable
+      && attributes.DocumentExpirationNotApplicable.applicable === false)
+      || !options.requireRelationshipMaritalForeignBornDocExpiration
```
Comment (suzubara) [inline src/models/foreignBornDocument.js]: can we name options independently of the context from where they're being set (i.e. `requireForeignBornDocExpiration`)?

**Comment by christopherhuii** [comment]
@suzubara All issues have been fixed. However, I want a second pair of eyes on Relatives. The relatives branching is a little confusing and it'd probably be helpful to review it together.

**Comment by ryanhofdotgov** [comment]
@christopherhuii I'm going to ask that @suzubara review this instead, as I think it deserves more detailed attention/familiarity than I will bring to bear.

File: src/models/relative.js [hunk 2]
```diff
@@ -5,9 +5,13 @@ import usAddress from 'models/shared/locations/usAddress'
 import birthplaceWithoutCounty from 'models/shared/locations/birthplaceWithoutCounty'
 import physicalAddress from 'models/shared/physicalAddress'
 import { hasYesOrNo } from 'models/validate'
+import * as formTypes from 'constants/formTypes'
```
Comment (suzubara) [inline src/models/relative.js]: doesn't look like `formTypes` ends up being used in this file

File: src/validators/foreignborndocument.js [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
 
-export const validateForeignBornDocument = data => (
-  validateModel(data, foreignBornDocument) === true
-)
+export const validateForeignBornDocument = (data, formType = formTypes.SF86) => {
+  const isForeignBornDocExpirationRequired = requireRelationshipMaritalForeignBornDocExpiration(formType)
+  return validateModel(
+    data,
+    foreignBornDocument,
+    { requireForeignBornDocExpiration: isForeignBornDocExpirationRequired }
```
Comment (suzubara) [inline src/validators/foreignborndocument.js]: I don't think this option is being passed down correctly from the marital or civil union validator... on the SF-86, the section shows as valid in the sidebar even if there is no value in the foreign born doc expiration date.

File: src/models/relative.js [hunk 4]
```diff
@@ -26,29 +30,51 @@ export const isLiving = attributes => (
...
+  && attributes.AlternateAddress.HasDifferentAddress
+  && attributes.AlternateAddress.HasDifferentAddress === 'Yes'
+)
+
+export const requireCitizenshipDocumentation = (attributes, options) => {
+  if (options.requireRelationshipRelativesForeignBornDoc) {
+    return !!((
+      isCitizen(attributes)
+      && !wasBornInUS(attributes)
+      && !isLiving(attributes)
```
Comment (suzubara) [inline src/models/relative.js]: is this correct - the documentation is required even if the relative is deceased?

Comment (christopherhuii) [reply src/models/relative.js]: I believe so. That's how the form tracking doc reads and 18.11 in the validation matrix.

**Pull request merged**